docs/home/api-info/webhooks.mdx

```diff
@@ -92,16 +94,18 @@ else:
 ```
 
 <Warning>
-The previous method of authenticating webhooks, comparing your webhook key with the `x-webhook-key` in each request's HTTP header, is being deprecated on **December 9th, 2023**. 
-Please update your implementation to use the `x-metriport-signature` header for authentication.
+  The previous method of authenticating webhooks, comparing your webhook key with the
+  `x-webhook-key` in each request's HTTP header, is being deprecated on **December 9th, 2023**.
```

Comment:
let's update the date, or take this out?

we're a bit past `December 9th, 2023` 😛

Reply:
Missed that, removed!